libs/nspr/patches/002-Add-nios2-support.patch

```diff
@@ -0,0 +1,102 @@
+From 95bda64fb4cf1825fea745e918cfe8202843f0ba Mon Sep 17 00:00:00 2001
+From: Marek Vasut <marex@denx.de>
+Date: Sat, 30 Jan 2016 07:18:02 +0100
+Subject: [PATCH] Add nios2 support
+
+Add support for the nios2 CPU.
```

Comment:
Is nios even an openwrt target?  this is a 2016 patch?  Are you just blindly copying patches from some other distro?

Reply:
some of them were from openembedded, I could drop this one

Reply:
Please do